scss/titles.scss

```diff
@@ -0,0 +1,16 @@
+.title {
+  align-items: center;
+  color: #000;
+  display: flex;
+  font-size: 1rem;
+  padding: 10px;  
+
+  &::before {
+    background: url('assets/triangle.svg') no-repeat right;
+    content: '';
+    margin-right: -10px;
```

Comment:
Use `position: absolute` instead. Then you will have to add `left or right and top`. You will have to add `position: relative` to the parent